src/components/homePage/Hero.vue

```diff
@@ -17,6 +17,9 @@
 		<form action="https://cutc.us10.list-manage.com/subscribe/post?u=73d566326ae6bb629b42fb5fd&amp;id=e28a4a51fe" method="post" 
 				target="_blank" novalidate>
 			<input type="email" name="EMAIL" placeholder="Enter your email for updates!" required>
+			<button>
+				<img src="../../assets/Group.svg">
+			</button>
```

Comment:
I think it'd be better if this was a input of type="submit" or something, but I'm not entirely sure. What are your thoughts on this @namansood